### PR TITLE
Moved Envoys.reload to async

### PR DESCRIPTION
**Pull request by WaterArchery**
SlimeWorldManager loads and unloads worlds lots of times. So, using io operations halts the few server ticks. To prevent that, I moved the reload function to async.

### CODE REVIEW

File: src/main/java/com/artillexstudios/axenvoy/envoy/Envoys.java [hunk 1]
```diff
@@ -18,47 +20,52 @@ public class Envoys {
...
-        for (int i = 0; i < chestSize; i++) {
-            Envoy envoy = envoyTypes.get(i);
-
-            if (removedTypes.contains(envoy)) {
-                envoy.stop();
-            } else {
-                envoy.reload();
-            }
-        }
+        }.runTaskAsynchronously(AxEnvoyPlugin.getInstance());
```
Comment (AverageGithub) [inline src/main/java/com/artillexstudios/axenvoy/envoy/Envoys.java:68]: This approach won't cut it, as we support Folia. Use the Scheduler.get().runAsync(Runnable) method.

**Review by AverageGithub — CHANGES_REQUESTED** [review]
Thank you for your contribution! I requested a change that should be pretty easy to resolve!

**Comment by WaterArchery** [comment]
I implemented your change request.

**Review by AverageGithub — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**